src/data/fintype/basic.lean

```diff
@@ -1043,6 +1043,10 @@ by { ext, simp only [set.mem_empty_eq, set.mem_to_finset, not_mem_empty] }
   s.to_finset = ∅ ↔ s = ∅ :=
 by simp [ext_iff, set.ext_iff]
 
+@[simp] lemma set.to_finset_range [decidable_eq α] [fintype β] (f : β → α) [fintype (set.range f)] :
+  set.to_finset (set.range f : set α) = finset.univ.image f :=
```

Comment:
```suggestion
  set.to_finset (set.range f) = finset.univ.image f :=
```

any reason to make the coe explicit? otherwise looks good! (i still don't know if I'm allowed to waltz into other people's PRs and say LGTM or not, lol)

Reply:
Thanks, copied from something that said `set.univ` and I wasn't paying attention